server/package.json

```diff
@@ -16,6 +16,7 @@
     "http-errors": "~1.6.3",
     "joi": "^14.3.1",
     "jsonwebtoken": "^8.5.1",
+    "lodash": "^4.17.14",
```

Comment:
Can we also update lodash on the front-end ^4.17.14 and run yarn install. lower versions actually had security problem